### DISCUSSION TITLE
No such image error after scanning local image

### DISCUSSION THREAD

**Discussion by daniel-adamek-eit** [aquasecurity/trivy#4967] [Bugs]
### Description

I have this script inside my azure pipeline:

      - script: |
          docker images
          trivy image ${{ parameters.serviceName }}:$(version)
        displayName: trivy-scan

docker images produces this output: 

```
REPOSITORY                       TAG         IMAGE ID      CREATED       SIZE
***/image-name                 0.0.5       <someID>     53 years ago  372 MB
```

Trivy scan produce this ERRORS:

```
2023-08-08T14:41:56.312Z	FATAL	image scan error: scan error: unable to initialize a scanner: unable to initialize a docker scanner: 4 errors occurred:
	* unable to inspect the image (image-name:0.0.5): Error response from daemon: No such image: image-name:0.0.5
	* containerd socket not found: /run/containerd/containerd.sock
	* unable to initialize Podman client: no podman socket found: stat podman/podman.sock: no such file or directory
	* GET https://index.docker.io/v2/library/image-name/manifests/0.0.5: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:library/image-name Type:repository]]
```

### Desired Behavior

Scan of image without error

### Actual Behavior

Errors: image scan error: scan error: unable to initialize a scanner: unable to initialize a docker scanner: 4 errors occurred:

### Reproduction Steps

```bash
1.install trivy on image which is used as a agent for pipeline jobs
2.run trivy scan on local image
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
2023-08-08T14:41:56.312Z	FATAL	image scan error: scan error: unable to initialize a scanner: unable to initialize a docker scanner: 4 errors occurred:
	* unable to inspect the image (image-name:0.0.5): Error response from daemon: No such image: image-name:0.0.5
	* containerd socket not found: /run/containerd/containerd.sock
	* unable to initialize Podman client: no podman socket found: stat podman/podman.sock: no such file or directory
	* GET https://index.docker.io/v2/library/image-name/manifests/0.0.5: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:library/image-name Type:repository]]
```

### Operating System

Ubuntu

### Version

```bash
Version: 0.44.0
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
I guess you just specified a wrong image name. `image-name:0.0.5` should be `***/image-name:0.0.5`.

**Reply by daniel-adamek-eit** [reply]
I tried that and now It gives me only one ERROR:

`FATAL	image scan error: scan error: unable to initialize a scanner: unable to initialize a docker scanner: failed to parse the image name: could not parse reference: ***/image-name:0.0.6`

Why does it mention "**initialize a docker scanner**" when Im not using trivy from docker Image? Trivy is already installed using apt-get.

**Reply by DmitriyLewen** [reply]
Hello @ser789654 
`***/image-name:0.0.6` is that typo? I wrote about `0.0.5` version.
